packages/.eslintrc.json

```
@@ -83,6 +83,15 @@
         ]
     },
     "overrides": [
+        {
+            "files": [
+                "**/test/**/*.test.{js,ts,jsx,tsx}",
+                "**/stories/**/*.{js,ts,jsx,tsx}"
```

This looks great but do we want to scope these a little more?

```suggestion
                "{packages,tools}/*/test/*.test.ts",
                "{packages,tools}/*/stories/*.ts"
```

To make sure we're not ignoring assets we want to validate?

---

Oh this is along the lines of my comment but I like this recommendation better than mine.